extensions/skins/conversation_v1/static/conversation.css

```diff
@@ -27,162 +32,288 @@ html, body {
   All classes below should start with .conversation-skin
  */
 
-.conversation-skin-cards-container {
-  margin-bottom: 50px;
-}
-
-.conversation-skin-exploration-intro-card {
-  background-color: rgb(255,255,255);
-  font-family: 'Roboto';
-  margin: 20px auto;
-  max-height: 567px;
-  max-width: 560px;
-  padding: 0px;
-  text-align: center;
+/* Overwrite the default color for the 'Loading...' message. */
+.oppia-loading-fullpage {
+  color: #fff;
 }
 
-
-.conversation-skin-left-gadget-panel {
-  margin-right: 285px;
-  max-width: 200px;
+.conversation-skin-progress-dots {
+  left: 0;
+  margin: 0 auto;
   position: fixed;
-  right: 50%;
-  top: 76px;
+  right: 0;
+  top: 56px;
+  z-index: 100;
 }
 
-.conversation-skin-right-gadget-panel {
-  margin-left: 285px;
-  max-width: 200px;
-  position: fixed;
-  left: 50%;
-  top: 76px;
+.conversation-skin-initial-oppia-avatar,
+.conversation-skin-oppia-avatar,
+.conversation-skin-user-avatar {
+  position: absolute;
 }
-
-.conversation-skin-card-row-container {
-  margin-bottom: 17px;
-  table-layout: fixed;
-  width: 100%;
+.conversation-skin-initial-oppia-avatar {
+  height: 36px;
+  left: -18px;
+  top: 30px;
+  width: 36px;
 }
-
-.conversation-skin-card-row {
-  margin-bottom: 20px;
-  margin-top: 20px;
-  width: 100%;
+.conversation-skin-oppia-avatar {
+  height: 36px;
+  left: -18px;
+  top: -9px;
+  width: 36px;
 }
-
-.conversation-skin-row-avatar {
-  padding-right: 10px;
-  width: 30px;
+.conversation-skin-user-avatar {
+  height: 36px;
+  left: -18px;
+  top: 12px;
+  width: 36px;
 }
 
-.conversation-skin-row-avatar-img {
-  height: 24px;
-  max-width: 24px;
+.conversation-skin-responses-dropdown-container {
+  border-bottom: 1px solid #ccc;
+  cursor: pointer;
+  line-height: 0.1em;
+  margin: 0 auto 30px auto;
+  text-align: center;
+  width: 90%;
+}
+.conversation-skin-responses-dropdown-text {
+  background-color: #fff;
+  color: #888;
+  font-size: 0.8em;
+  padding: 0 10px;
 }
 
-.conversation-skin-exploration-header {
-  background: linear-gradient(to bottom, rgba(255, 255, 255, 0.9), rgba(255, 255, 255, 1));
-  font-size: 1.25em;
-  text-align: center;
-  height: 30%;
-  margin-top: -13.9%;
-  padding-bottom: 1%;
-  padding-top: 1%;
-  position: relative;
+.conversation-skin-main-gadget-panel {
```

Comment:
Could you add a TODO for it to be filled in, then? I just want to make sure it isn't lost to the ages for some reason.